### PR TITLE
adding leave a message section

### PR DESCRIPTION
**Pull request by JimCarlier**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -246,6 +247,24 @@ <h4>UX Designer</h4>
           </div>
         </div>
       </section>
```
Comment (glgoose) [inline index.html:250]: space after section (to visually separate them like rest of document)

File: index.html [hunk 2]
```diff
@@ -246,6 +247,24 @@ <h4>UX Designer</h4>
...
+      <section class="leave-a-message">
+        <h2><i class="lni lni-envelope"></i><span>Leave</span> a Message.</h2>
```
Comment (glgoose) [inline index.html]: `<strong>` would be better suited than `<span>`

File: index.html [hunk 3]
```diff
@@ -246,6 +247,24 @@ <h4>UX Designer</h4>
           </div>
         </div>
       </section>
+      <section class="leave-a-message">
```
Comment (glgoose) [inline index.html]: class name `"message"` for short would be better, the classes are for styling an don't need to be very descriptive
e.g. `leave-message` would be good for `id` if use as an anchor link

File: index.html [hunk 4]
```diff
@@ -246,6 +247,24 @@ <h4>UX Designer</h4>
...
+        <h2><i class="lni lni-envelope"></i><span>Leave</span> a Message.</h2>
+        <form>
+          <div class="leave-a-message__div">
```
Comment (glgoose) [inline index.html]: `message__div`

**Review by glgoose — CHANGES_REQUESTED** [review]

**Review by jacopobacci — COMMENTED** [review]
agree with glenn's changes requests

**Pull request merged**